### DISCUSSION TITLE
Error in myloader

### DISCUSSION THREAD

**Discussion by borgesbrrs** [mydumper/mydumper#1149] [General]
Hi guys,
I make withou problems an backup ok, but when I will make the restore I receive this error.

ubuntu@ip-172-30-84-89:~$ myloader --version
myloader 0.9.5, built against MySQL 8.0.19

ubuntu@ip-172-30-84-89:~$ myloader -h xxx.clwigc1omkul.us-east-1.rds.amazonaws.com -u root --password='teste123' --database  test -d /data/
Segmentation fault (core dumped)

So 
I folow those doc to make this procedure.
https://repost.aws/pt/knowledge-center/rds-mysql-cross-region-replica

**Comment by davidducos** [comment]
@borgesbrrs please, use a newer version.

**Reply by borgesbrrs** [reply]
aaaaaaa man, thanks David  :)
